packages/documentation/docs/controls/_date-input_styleguide.mdx

```diff
@@ -0,0 +1,69 @@
+The date input component enables users to enter and select a date in a standardized format. We typically use this component in forms, calendars, and scheduling tools to ensure consistent and accurate date entries.
+
+![Date input overview](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3629-6200&t=ADQCetGKOEH1WG2r-4)
+1. Label
+2. Required field indicator
+3. Current value
+4. Calendar icon button
+5. Input field
+6. Date dropdown
+7. Month and year selection
+8. Weekdays
+9. Week numbers 
+10. Indicator for current day
+11. Indicator for selected day
+
+## Options
+
+- **Label**: See [form field](./forms/forms-field.md)
+- **Value**: See [form field](./forms/forms-field.md)
+- **Required**: See [form field](./forms/forms-field.md)
```

Comment:
we haven't had this on the other markdowns. should we add it everywhere?

Reply:
+ same for a couple other listed fields here

Reply:
Yes, we should check the options mentioned in the description. Kept it here and created a ticket.